IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -232,7 +317,45 @@ def describe_screen(screen_id):
     return plate_results_df
 
 
-# In[10]:
+# In[22]:
+
+
+def collect_metadata(idr_name, values_list):
+    """Data collection and saving pipeline"""
+    data_dir = pathlib.Path("../data/metadata")
+    if data_dir.exists() == False:
+        os.mkdir(data_dir)
+
+    # Extract names and values
+    screen_id = values_list[0]
+    imaging_method = values_list[1]
+    sample = values_list[2]
+    split_name = idr_name.split("/")
+    study_name = split_name[0]
+    screen_name = split_name[1]
+
+    # Make directories
+    study_dir = pathlib.Path(data_dir, study_name)
+    screen_dir = pathlib.Path(study_dir, screen_name)
+    if screen_dir.exists() == False:
+        os.makedirs(screen_dir, exist_ok=True)
```

Comment:
same pathlib comment as above

Reply:
Addressed in same commit as above: ade7519d00aa36e6d4211189cdff064fdecc89b7